### DISCUSSION TITLE
When opening the dialog on a smaller screen, it scrolls to top

### DISCUSSION THREAD

**Discussion by SincerelyFaust** [tailwindlabs/headlessui#2472] [Help]
Basically when opening the dialog on bigger screens it works as usual but on smaller screens it opens the dialog as it should but in the background you can see the rest of the website scrolling to the top.

**Comment by SincerelyFaust** [comment]
I have figured out why this happens in my case, it is due to `scroll-behavior: smooth`...

**Reply by robinsimonklein** [reply]
This is probably related to https://github.com/tailwindlabs/headlessui/discussions/2388. 
In my case, I found a workaround to keep the scroll-behavior on my page when the dialog isn't open : https://github.com/tailwindlabs/headlessui/discussions/2388#discussioncomment-5817530

Maybe this might help you :)
